src/matvis/core/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""Core functionality of the matvis package.
+
+This sub-package defines template routines for the core functionality of the
+algorithm. These routines are then implemented in the ``cpu`` and ``gpu``
+sub-packages.
+"""
+
+import numpy as np
+
+
+def _validate_inputs(
```

Comment:
Just curious why keeping this function in the __init__?

Reply:
really just because I couldn't think of another place to put it. I could change it if you have an idea.

Reply:
Ah. let's just keep this for now.